src/codegen/doc/mod.rs

```diff
@@ -517,13 +561,34 @@ where
             }
         }
 
-        if let Some(ref doc) = fn_.ret().doc {
+        let out_parameters: Vec<_> = no_array_length_params
+            .iter()
+            .filter(|param| {
+                param.direction == ParameterDirection::Out
+                    && !(param.name == "error" && param.c_type == "GError**")
+            })
+            .collect();
+
+        if fn_.ret().doc.is_some() || !out_parameters.is_empty() {
```

Comment:
Yeah I was wondering about this one already, now the out parameters neatly show up even if there is no return doc :)